File: javascript/moderation_text_token_demo.js
/**
 *  图像内容检测服务token请求方式的使用示例
 */
var text = require("./ais_sdk/moderation_text");
var token = require("./ais_sdk/gettoken");
var utils = require("./ais_sdk/utils");

// 初始化服务的区域信息，目前支持华北-北京一(cn-north-1)、亚太-香港(ap-southeast-1)等区域信息
utils.initRegion("cn-north-1");

var username = "*******";        // 配置用户名
var domain_name = "*******";     // 配置用户名
var password = "*******";        // 密码

token.getToken(username, domain_name, password, function (token) {

    text.moderation_text(token, [{
            "text": "666666luo聊请+110亚砷酸钾六位qq，fuck666666666666666",
            "type": "content"}],
        ["ad", "abuse", "politics", "porn", "contraband"], function (result) {
            console.log(result);
        })

});

File: python3/moderation_text_token_demo.py
# -*- coding:utf-8 -*-
from ais_sdk.gettoken import get_token
from ais_sdk.moderation_text import moderation_text

if __name__ == '__main__':
    #
    # access moderation text enhance,posy data by token
    #
    user_name = '******'
    password = '******'
    account_name = '******'  # the same as user_name in commonly use

    token = get_token(user_name, password, account_name)

    # call interface use the text
    result = moderation_text(token, '666666luo聊请+110亚砷酸钾六位qq，fuck666666666666666', 'content')
    print(result)